src/components/__tests__/ProgressCircle.spec.ts

```diff
@@ -119,12 +120,6 @@ describe("ProgressCircle", () => {
         expect(progressCircle.animate).toHaveBeenCalledWith(1);
     });
 
-    it("has the class widget-progress-circle-negative when the value is less than zero", () => {
```

Comment:
You just can not remove a feature. without consulting the client. This is a breaking change. May be the boostrap style for negative value should be configured too, please check.